### PR TITLE
[Chore|SDE|Helm|1.0.0] Helm charts update for AppV 2.3.1

### LINKED ISSUE #23: [TRG 5.07] postgresql version is not on the aligned version
Hi @adityagajbhiye9,

as discussed on the mailing list [the teams decided to align](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-07#aligning-dependency-versions) the postgresql version to 15.x.

Please update/migrate your Bitnami chart version 12.x

### PR DESCRIPTION
**Pull request by adityagajbhiye9**
## Description
Fixed 
#23  Postgres version aligned to v15.
#22 Default pass word removed from values.yaml
 
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by adityagajbhiye9** [comment]
@almadigabor Could you please Review it and merge PR  ?
@sachinargade123  Please check and review it.

**Review by sachinargade123 — APPROVED** [review]
Changes looks ok

File: charts/simpledataexchanger/values.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ sdepostgresql:
   enabled: true
   fullnameOverride: "product-sde-postgres"
   auth:
-    postgresPassword: "default"
+    postgresPassword: ""
```
Comment (almadigabor) [inline charts/simpledataexchanger/values.yaml:29]: Is there a function that generates a random password when this is left empty?

**Review by almadigabor — COMMENTED** [review]
See my comment below

**Comment by almadigabor** [comment]
Also please take a look at the failed helm test workflow.

File: charts/simpledataexchanger/values.yaml (hunk 1 above)
Comment (adityagajbhiye9) [reply charts/simpledataexchanger/values.yaml:29]: yes, It will generate random password.

**Comment by adityagajbhiye9** [comment]
> Also please take a look at the failed helm test workflow.

Images hasn't been created that why it is failing.

File: charts/simpledataexchanger/values.yaml [hunk 2]
```diff
@@ -58,7 +58,7 @@ backend:
       drop:
         - ALL
     runAsNonRoot: true
-    runAsUser: 1001
+    runAsUser: 101
```
Comment (sachinargade123) [inline charts/simpledataexchanger/values.yaml:61]: please change this run user

**Pull request closed without merging**